pkg/daemon/controller_linux.go

```diff
@@ -574,7 +574,9 @@ func (c *Controller) handlePod(key string) error {
 
 	// set default nic bandwidth
 	ifaceID := ovs.PodNameToPortName(podName, pod.Namespace, util.OvnProvider)
-	err = ovs.SetInterfaceBandwidth(podName, pod.Namespace, ifaceID, pod.Annotations[util.EgressRateAnnotation], pod.Annotations[util.IngressRateAnnotation])
+	ovsIngress := pod.Annotations[util.EgressRateAnnotation]
+	ovsEgress := pod.Annotations[util.IngressRateAnnotation]
```

Comment:
We can add a comment here to make it clear why `ovsIngress` is set to pod egress rate.